content/code/rest/batch-publish.code

```diff
@@ -0,0 +1,90 @@
+[--- Javascript ---]
+var ablyRest = new Ably.Rest({ key: 'xVLyHw._BYOCg:MjTf0KEZkSBj9nZj' }),
```

Comment:
Is this API key meant be hardcoded in?  I can't recall how the others are done...